examples/cdk/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "cdk-app",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Is this expected to follow same version of the rest of utils?

Reply:
done in https://github.com/awslabs/aws-lambda-powertools-typescript/pull/446/commits/2852495933b01ababd5092c057e2a548711b7821